packages/design-system/src/Link/Link.stories.tsx

```diff
@@ -52,7 +52,7 @@ InternalLink.args = {
 
 export const ExternalLink = Template.bind({});
 ExternalLink.args = {
-  to: "https://appsmith.com",
+  to: "/signup/test",
```

Comment:
Revert this, and for internal link let's set target to default to "_self" if there is no prop passed.